Sources/EdgeNetworkHandlers/EdgeHitProcessor.swift

```diff
@@ -129,6 +131,17 @@ class EdgeHitProcessor: HitProcessing {
         }
     }
 
+    /// Builds the endpoint based on the provided config info and `EdgeRequestType`
+    /// - Parameters:
+    ///   - config: configuration data, used to extract the environment and the custom domain, if any
+    ///   - requestType: the `EdgeRequestType`
+    private func buildEndpoint(config: [String: String], requestType: EdgeRequestType) -> EdgeEndpoint {
```

Comment:
nit: buildEdgeEndpoint and we can drop edge from the function parameters.

Reply:
params seem alright, is your comment to rename buildEndpoint -> buildEdgeEndpoint?

Reply:
yes